idl/response/foxsi_get_output_2d_image.pro

```diff
@@ -138,9 +138,13 @@ ENDIF
 ;;; equal to the value of the px keyword (default = 3'' per pixel)
 ;;; The centre of the map is preserved as the centre of the source image
 
+;rebinned_convolved_map = make_map(rebinned_convolved_array, dx = pix_size, dy = pix_size,   $
+;                         xc = source_map.xc, yc = source_map.yc, id = STRCOMPRESS(          $
+;                         'Rebinned_Convolved_Map_Pixel_Size:'+string(pix_size),/REMOVE_AL))
+
+; Version with simpler map ID:
```

Comment:
There's no reason to preserve the earlier code in comments instead of just deleting it.  For future maintenance and extendability, it might be nice to break out the assignment of `id` from the function call so that the user could toggle between options or something like that.